libs/langchain/langchain/llms/azureml_endpoint.py

```diff
@@ -92,44 +116,92 @@ def format_response_payload(self, output: bytes) -> str:
         """
 
 
+class GPT2ContentFormatter(ContentFormatterBase):
+    """Content handler for GPT2"""
+
+    def format_request_payload(self, prompt: str, model_kwargs: Dict) -> bytes:
+        prompt = ContentFormatterBase.escape_special_characters(prompt)
+        request_payload = json.dumps(
+            {"inputs": {"input_string": [f'"{prompt}"']}, "parameters": model_kwargs}
+        )
+        return str.encode(request_payload)
+
+    def format_response_payload(self, output: bytes) -> str:
+        return json.loads(output)[0]["0"]
+
+
 class OSSContentFormatter(ContentFormatterBase):
```

Comment:
you can also just do `OSSContentFormatter(GPT2ContentFormatter)`, and override init to throw a warning before instantiating

Reply:
Is the implementation fine for pr though? or should I change it

Reply:
Changed `OSSContentFormatter` to inherit `GPT2ContentFormatter`